app/image/image.py

```diff
@@ -0,0 +1,44 @@
+"""Support for containing image."""
+
+import numpy
```

Comment:
why do you import `numpy`? It is not used anywhere

Reply:
It's used for retrieving information about width and height of an image in `width` and `height` properties. `processed_data` is a numpy array.

Reply:
OK, please document this (could be a docstring in the constructor)